packages/cloudflare/src/cli/build/patches/ast/vercel-og.ts

```diff
@@ -0,0 +1,63 @@
+import { SgNode } from "@ast-grep/napi";
+
+import { applyRule } from "./util.js";
+
+export const vercelOgImportRule = `
+rule:
+  pattern: $NODE
+  kind: string
+  regex: "next/dist/compiled/@vercel/og/index.node.js"
+inside:
+  kind: arguments
+  inside:
+    kind: call_expression
+    stopBy: end
+    has:
+      field: function
+      regex: "import"
+
+fix: |-
+  "next/dist/compiled/@vercel/og/index.edge.js"
+`;
+
+/**
+ * Patches Node.js imports for the library to be Edge imports.
+ *
+ * @param root Root node.
+ * @returns Results of applying the rule.
+ */
+export function patchVercelOgImport(root: SgNode) {
+  return applyRule(vercelOgImportRule, root);
+}
+
+export const vercelOgFallbackFontRule = `
+rule:
+  kind: variable_declaration
+  all:
+    - has:
+        kind: variable_declarator
+        has:
+          kind: identifier
+          regex: ^fallbackFont$
+    - has:
+        kind: call_expression
+        pattern: fetch(new URL("$PATH", $$$REST))
+        stopBy: end
+
+fix: |-
+  async function getFallbackFont() {
+    return (await import("$PATH.bin")).default
```

Comment:
thinking more about it, I'm not sure we really need the `.bin`. Would you mind trying with that?

Reply:
You need to configure a custom loader for ttf files if you don't use .bin.

Reply:
Would you mind adding a comment in the code?